CONTRIBUTING.md

```diff
@@ -116,39 +116,57 @@ The current list of labels are [here][link_labels] and include:
 
 ## Making a change
 
-We appreciate all contributions to tedana, but those accepted fastest will follow a workflow similar to the following:
+We appreciate all contributions to `tedana`, but those accepted fastest will follow a workflow similar to the following:
 
-**1. Comment on an existing issue or open a new issue referencing your addition.**
+### 1. Comment on an existing issue or [open a new issue][link_createissue] referencing your addition
 
-This allows other members of the tedana development team to confirm that you aren't overlapping with work that's currently underway and that everyone is on the same page with the goal of the work you're going to carry out.
+This allows other members of the `tedana` development team to confirm that you aren't overlapping with work that's currently underway and that everyone is on the same page with the goal of the work you're going to carry out.
 
 [This blog][link_pushpullblog] is a nice explanation of why putting this work in up front is so useful to everyone involved.
 
-**2. [Fork][link_fork] the [tedana repository][link_tedana] to your profile.**
+### 2. [Fork][link_fork] the [tedana repository][link_tedana] to your GitHub profile
 
-This is now your own unique copy of tedana. Changes here won't effect anyone else's work, so it's a safe space to explore edits to the code!
+This is now your own unique and online copy of `tedana`. Changes here won't affect anyone else's work, so it's a safe space to explore edits to the code!
 
-Make sure to [keep your fork up to date][link_updateupstreamwiki] with the master repository.
+Remember to [clone your fork][link_clonerepo] of `tedana` to your local machine, which will allow you to make local changes to `tedana`.
 
-**3. Make the changes you've discussed.**
+Make sure to always [keep your fork up to date][link_updateupstreamwiki] with the master repository before and after making changes.
 
-Try to keep the changes focused. We've found that working on a [new branch][link_branches] makes it easier to keep your changes targeted.
+### 3. Run the developer setup
 
-When you're creating your pull request, please make sure to review the tedana [style conventions](#style-guide).
-
-Note that to test a change, you may need to set up your local repository to run a tedana workflow. 
+To test a change, you may need to set up your local repository to run a `tedana` workflow. 
 To do so, run
 ```
 python setup.py develop
```

Comment:
```suggestion
pip install -e
```

Reply:
Can you add the full command @jbteves? Probably the GitHub link...

Reply:
Added the dot since the next line says "from your repository."

Reply:
Why `pip install -e .` over `python setup.py develop`? I normally use `setup.py` myself.

Reply:
I just saw @jsheunis' comment in the corresponding issue. I assume that's why. Never mind!